redbot/cogs/mod/settings.py

```diff
@@ -370,6 +388,62 @@ async def dm(self, ctx: commands.Context, enabled: bool = None):
                 _("Bot will no longer attempt to send a DM to user before kick and ban.")
             )
 
+    @dm.command(name="banshowextrafield")
+    async def dm_banshowextrafield(self, ctx: commands.Context, enabled: bool = None):
+        """
+        Toggle whether to show an extra customizable field when banning. This is useful to add information such as a ban appeal link.
```

Comment:
```suggestion
        Toggle whether to show an extra customizable field when banning.

        This can be used to add additional information for the banned user, such as a ban appeal link.
```
Phrasing & formatting (should also be applied in the docs)